### PR TITLE
Update user.md

### PR DESCRIPTION
**Pull request by helenalt**
added a clarification about where bazel tests run and which are internal vs external tests

### CODE REVIEW

**Comment by helenalt** [comment]
updated ci doc to make explicit what tests are run in ci.bazel.io

**Comment by damienmg** [comment]
there is a problem with master, it is not in sync with googlesource...

File: docs/user.md [hunk 1]
```diff
@@ -18,6 +18,11 @@ Bazel CI, see the [project owner documentation](owner.md).
 ## Postsubmit <a name="postsubmit"></a>
 
 Every project that runs on Bazel CI is run on postsubmit.
+All test jobs are visible at http://ci.bazel.io/
```
Comment (damienmg) [inline docs/user.md]: I don't understand that section so I would recommend to remove it.

**Comment by damienmg** [comment]
jenkins: retest this please

**Comment by damienmg** [comment]
let's retry one more time: retest this please

**Pull request merged**